libraries/app/application.cpp

```diff
@@ -439,6 +439,13 @@ void application_impl::startup()
    if ( _options->count("enable-subscribe-to-all") )
       _app_options.enable_subscribe_to_all = _options->at( "enable-subscribe-to-all" ).as<bool>();
 
+   if ( _options->count("max-account-history-operations-limit") )
+   {
+      uint32_t max_acct_hist_limit = _options->at("max-account-history-operations-limit").as<uint32_t>();
+      _app_options.max_account_history_operations_limit = _options->at("max-account-history-operations-limit").as<uint32_t>();
```

Comment:
you are doing the same thing twice, `max_acct_hist_limit` is not used.

Reply:
default value of 100 should be added here. also, i am not sure if the option should work for ALL the hardcoded limits or only for the account history related. If it is for all hardcoded values inside the API then the name of the option is not very well picked.

Reply:
removed extra line at 444